mathics/core/streams.py

```diff
@@ -13,12 +13,13 @@
 import requests
 
 from mathics.core.util import canonic_filename
-from mathics.settings import ROOT_DIR
+from mathics.settings import ROOT_DIR, USER_PACKAGE_DIR
 
 HOME_DIR = osp.expanduser("~")
 PATH_VAR: List[str] = [
     ".",
     HOME_DIR,
+    USER_PACKAGE_DIR,
     osp.join(ROOT_DIR, "data"),
     osp.join(ROOT_DIR, "packages"),
```

Comment:
@mmatera should "packages" get renamed to "Packages" to be more conformant with WMA?

Reply:
Maybe this change helps the implementation of paclets.

Reply:
In order to be compatible with paclets, maybe the change is a good idea